src/index.html

```diff
@@ -30,11 +30,11 @@ <h4>Recent scores</h4>
         </div>
       </div>
       <div class="form-container">
-        <h4>Add your score</h4>
+        <h4 class="recent">Add your score</h4>
         <form action="#" class="d-flex form">
           <input type="text" class="name form-control mb-4" placeholder="Your name">
           <input type="number" class="score form-control mb-4" placeholder="your score">
-          <input type="submit" value="submit" id="submit" class="btn btn-success">
+          <input type="submit" value="submit" id="submit" class="sub btn btn-success">
```

Comment:
The data in the server is somehow unstructured, this time I have retrieved all the data so that you can see it. It is grouped. At first, I was remitting up to 100.